test/transform.js

```diff
@@ -23,7 +23,17 @@ test('transform adds sourcemap comment and uses cache on second time', function
 
     var es6ify = proxyquire('..', { './compile' : trackingCompile } )
 
-    var file = path.join(__dirname, '../example/src/features/iterators.js');
+    var sourceRoot = path.join(__dirname, '..', 'example', 'src');
+    var relPath = path.join('features', 'iterators.js');
+
+    var opts = {sourceRoot: sourceRoot};
+
+    es6ify = es6ify.configure(opts);
+
+    var file = path.join(sourceRoot, relPath);
+    if (! opts.sourceRoot) relPath = path.basename(file);
```

Comment:
`'if (! opts.sourceRoot)'.length === 22`
`'if (opts.sourceRoot === undefined)'.length === 34`

Unlike the `filePattern` case, this is one where there may be an argument for allowing `0` or `''` as the value of `sourceRoot`, but I disagree that it's better style when the distinction between `undefined` and falsy is irrelevant. If that's what you want that's what I'll do though.